src/constants/formatters.js

```diff
@@ -0,0 +1,9 @@
+import * as d3Format from 'd3-format';
```

Comment:
Yeah I told her to put them here, but you're probably right `src/utils` is a better place since they technically aren't constants.